content/vendor/guides/cli-quickstart.md

```diff
@@ -0,0 +1,441 @@
+
+On the next screen, you have the option of uploading a trusted cert and key. For production installations we recommend using a trusted cert. For this demo let's continue with the KOTS-generated self-signed cert. Click the "skip this step" button.
+
+![Console TLS](/images/guides/kots/admin-console-tls.png)
+
+Now the installation needs a license file to continue. Until this point, this server is just running Docker, Kubernetes, and the kotsadm containers. Once we put a license file on it the server will install our application. Click the Upload button and select your `.yaml` file to continue.
+
+![Upload License](/images/guides/kots/upload-license.png)
+
+The settings page is here with default configuration items. These can be specified in the `config.yaml` file.
+
+![Settings Page](/images/guides/kots/configuration.png)
+
+Preflight checks are designed to ensure this server has the minimum system and software requirements to run the application. Depending on your YAML in `preflight.yaml`, you may see some of the example preflight checks fail. If you have failing checks, you can click continue -- the UI will show a warning that will need to be dismissed before you can continue.
+
+![Preflight Checks](/images/guides/kots/preflight.png)
+
+Click the Application link on the top to see the application running. If you are still connected to this server over ssh, `kubectl get pods` will now show the example nginx service we just deployed.
+
+![Dashboard](/images/guides/kots/dashboard.png)
+
+On the nav bar, there's a link to the application page. Clicking that will show you the Kubernetes services that we just deployed.
+
+![Cluster](/images/guides/kots/application.png)
+
+### View the application
+
+If you used the default nginx application, you can view the application at `http://${INSTANCE_IP}/` with no port, and you should see a basic (perhaps familiar) nginx server running:
+
+![Cluster](/images/guides/kots/example-nginx.png)
+
+Next, we'll walk through creating and delivering an update to the application we just installed.
+
+* * *
+
+### Iterating
+
+From our local repo, we can update the nginx deployment to test a simple update to the application. If you're using your own YAML or helm chart, you can simulate an application update by adding an annotation or making any other superficial change.
+
+In the release YAML, find the nginx image to change. The line is in the `deployment.yaml` file and looks like:
+
+```yaml
+replicas: 1
+```
+
+Change the number to `2` or more.
+
+we can create a new release
+
+```shell script
+replicated release create \
+  --yaml-dir manifests \
+  --promote Unstable \
+  --version dev-2 \
+  --release-notes "my second release"
+```
+
+Alternatively, this might also be a good time to explore the `--auto` flag to populate values for Channel, Version, and Release Notes (note that this requires you to be working in a git repo).
```

Comment:
How do I explore the --auto flag? is there documentation, a command I can run to find more about it?

Reply:
Good question. We don't have docs on it yet but what if we swap out `explore` with `try out`

```suggestion
Alternatively, this might also be a good time to try out the `--auto` flag to populate values for Channel, Version, and Release Notes based on your current git status (note that this requires you to be working in a git repo).
```

Reply:
I think the 'based on your current git status' helps... what I guess really wanted to know was why this requires to be working in a git repo.